src/pages/IstioConfigDetails/IstioConfigDetailsPage.tsx

```diff
@@ -233,27 +254,23 @@ class IstioConfigDetailsPage extends React.Component<RouteComponentProps<IstioCo
 
     return (
       <div className="container-fluid container-cards-pf">
-        <Row className="row-cards-pf">
```

Comment:
We should remove the className of pf3 styles

Reply:
Do you mind classes from the div right above? Because the ones from the Row are being deleted in this PR.

Regarding the classes in the div above, I see they are used in all the Detail pages: overview, app, workload, service and Istio Config. Perhaps we should leave this task to https://github.com/kiali/kiali/issues/1600 ?

Reply:
Yep...I was thinking in the grey color background Jira